openedx/core/djangoapps/user_authn/views/tests/test_login.py

```diff
@@ -666,10 +666,14 @@ def test_login_form(self):
         response = self.client.get(self.url, content_type="application/json")
         self.assertHttpOK(response)
 
+        # TODO: ARCH-1253: LoginSession GET returns a form that temporarily points to a different login URL for POST
+        # until we complete the roll-out/transition to the login_user response without the shim.
```

Comment:
Nit: Explain what we are doing in the meantime `until we ...`.
For example: `Use the POST login_api endpoint until we ...`

Reply:
Updated.